src/assets/js/boot-options.mjs

```diff
@@ -1,5 +1,55 @@
-'use strict';
+/* global browser */
 
-import {initOptions} from "@/options.mjs";
+import ReactDOM from 'react-dom/client';
+import React from 'react';
+import { useEffect } from 'react';
+import {
+  Provider,
+  useSelector,
+  useDispatch,
+} from 'react-redux';
 
-initOptions();
+import 'bootstrap/scss/bootstrap.scss';
+import '@/scss/style.scss';
+
+import { SignIn } from '@/components/options/sign-in.jsx';
+import { SignedIn } from '@/components/options/signed-in.jsx';
```

Comment:
For js/jsx files, you can optionally avoid adding extensions.

import { SignIn } from '@/components/options/sign-in';

should be good enough.